### PR TITLE
notice of verification email - i136

### PR DESCRIPTION
**Pull request by JamieC89**
Notify user about email verification #136

Created a new modal of notice type -
Pops up with some text and disables the standard 'close' event listeners of other modals so that the user has to click the respective button to show they've read the message. 
Did this by cloning the body node so that the original event listeners are removed.

Also a quick fix to the max height of the modals - the 'delete account' modal on the profile page was looking out of sorts so this seemed like a simple fix - tried on mozilla & chrome.

### CODE REVIEW

File: src/js/login.js [hunk 1]
```diff
@@ -44,12 +48,15 @@ $(document).ready(function() {
...
         })
-    }
+    }   
 
     function register(email, passw) {
         var req = window.user.register(email, passw)
 
-        req.done(xhr => window.location = "profile.html")
+        notice = "To get full functionality from this website you need to verify your account. An email has been sent to your registered address, please follow the included instructions to complete the process."        
+        req.done(xhr => {
```
Comment (smeets) [inline src/js/login.js]: +1

File: src/js/modal.js [hunk 2]
```diff
@@ -777,6 +778,34 @@ $(document).ready(function() {
 		document.body.appendChild(modal)
 	}
 
+	//simple modal with text, that user has to click continue to proceed.
+     modals.noticeModal = function(title, desc, method) {
+ 	      var confirmBtn = el('button#continue.btn', ['continue'])
+ 	      var modal = el('div#modalNotice.modal.notice', [
```
Comment (smeets) [inline src/js/modal.js]: Any specific reason for `#modalNotice` instead of `#modal`?

File: src/js/modal.js [hunk 3]
```diff
@@ -777,6 +778,34 @@ $(document).ready(function() {
...
+ 	              	el('div.{overflow-x: visible};', [
```
Comment (smeets) [inline src/js/modal.js]: Styling as a class?

File: src/js/modal.js [hunk 4]
```diff
@@ -777,6 +778,34 @@ $(document).ready(function() {
...
+ 	                el("div.modal-header-title", [title]),
+ 	                el("div#bottom-divider.modal-divider"),
+ 	                el("div", [desc]),
+ 	                confirmBtn
+ 	            ])
+ 	          ])
+
+ 	    //removes escape event listeners so user can only press continue to proceed.
+ 	   	var old_el = document.body
```
Comment (smeets) [inline src/js/modal.js]: That's a pretty smart idea. Maybe add a note somewhere that calling this function probably breaks all subsequent modals

Comment (smeets) [reply src/js/modal.js]: Or you could add an exclusion on the ESC keydown listener

File: src/js/modal.js [hunk 5]
```diff
@@ -777,6 +778,34 @@ $(document).ready(function() {
 		document.body.appendChild(modal)
 	}
 
+	//simple modal with text, that user has to click continue to proceed.
+     modals.noticeModal = function(title, desc, method) {
```
Comment (smeets) [inline src/js/modal.js]: Can you move this to `js/components/notice-modal.js` as a standalone file?

**Comment by JamieC89** [comment]
changes made:
Notice modal is now a standalone file in components, I didn't use the promise function as the functionality is quite straightforward but maybe that should be added.
click off modal & esc events are cancelled by removeEvent...

I put in the css style where it 'appears' onto the screen. Maybe we should add this in for the other modals?

**Comment by smeets** [comment]
I'll try and merge this tomorrow

**Pull request merged**